diff -Nru libwibble/wibble/tests/test_buffer.cpp libwibble/wibble/tests/test_buffer.cpp
--- libwibble/wibble/tests/test_buffer.cpp	2006-09-15 23:59:02.000000000 +0100
+++ libwibble/wibble/tests/test_buffer.cpp	2007-09-10 13:19:46.000000000 +0100
@@ -1,6 +1,8 @@
 #include <wibble/config.h>
 #include <wibble/sys/buffer.h>
 
+#include <cstring>
+
 using namespace std;
 using namespace wibble::sys;
 
diff -Nru libwibble/wibble/tests/test_netbuffer.cpp libwibble/wibble/tests/test_netbuffer.cpp
--- libwibble/wibble/tests/test_netbuffer.cpp	2006-09-15 23:59:02.000000000 +0100
+++ libwibble/wibble/tests/test_netbuffer.cpp	2007-09-10 13:19:05.000000000 +0100
@@ -1,6 +1,8 @@
 #include <wibble/config.h>
 #include <wibble/sys/netbuffer.h>
 
+#include <cstring>
+
 using namespace std;
 using namespace wibble::sys;
 
